### DISCUSSION TITLE
Idea: Enable systematic "early filtering"

### DISCUSSION THREAD

**Discussion by shaib** [pretalx/pretalx#1798] [Ideas]
Hi,

Bringing this up here so it isn't lost and forgotten later:

We wanted to reduce the load on reviewers, so after every submission had at least four reviews, we went and marked all the submissions that had at least one -1 and no +1 as "pending not accepted", and told reviewers to focus on what's left. Seems like an idea worth supporting (under some generalization) with a management command, if not UI.

**Comment by rixx** [comment]
Hmmmm, I understand wanting to automate this, but at the same time, I feel like it would be very hard to have a good interface for this, as there are enough use cases to basically want a DSL or just the Python shell.

However, plugins can add their own management commands, so a very tiny plugin with just a custom management command might be the easiest solution here!
